test/scripts/hack_task_test.rb

```diff
@@ -0,0 +1,68 @@
+require "test_helper"
+
+class HackTaskTest < ActiveSupport::TestCase
+  # TODO: Try to get these values dynamically.
+  FRAMEWORK_PACKAGES = [
+    "bullet_train",
+    "bullet_train-api",
+    "bullet_train-fields",
+    "bullet_train-has_uuid",
+    "bullet_train-incoming_webhooks",
+    "bullet_train-integrations",
+    "bullet_train-integrations-stripe",
+    "bullet_train-obfuscates_id",
+    "bullet_train-outgoing_webhooks",
+
+    # TODO: We need to add this with bin/hack --link
+    # "bullet_train-roles",
+
+    "bullet_train-scope_questions",
+    "bullet_train-scope_validator",
+    "bullet_train-sortable",
+    "bullet_train-super_load_and_authorize_resource",
+    "bullet_train-super_scaffolding",
+    "bullet_train-themes",
+    "bullet_train-themes-light",
+    "bullet_train-themes-tailwind_css",
+  ]
```

Comment:
Instead of maintaining this list it might suffice to just spot check two specific packages. For instance we could theoretically just test `bullet_train-api` as being representative of gems that _are_ listed in the `Gemfile` and test `bullet_train-fields` as being representative of gems that _are not_ listed in the `Gemfile`.